app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  after_save :update_post_counter
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_post_counter
```

Comment:
- [ ] [OPTIONAL]
Again, let's mark `update_post_count` as _private_ after implementing the `after_save` method.